mocks/routes/powerbrowser.js

```diff
@@ -22,16 +23,22 @@ module.exports = [
     ],
   },
   {
-    id: 'post-powerbrowser-search-person',
+    id: 'post-powerbrowser-search-requests',
     url: `${settings.MOCK_BASE_PATH}/powerbrowser/SearchRequest`,
     method: 'POST',
+    delay: 5000,
     variants: [
       {
         id: 'standard',
-        type: 'json',
+        type: 'middleware',
         options: {
-          status: 200,
-          body: BB_SEARCH_PERSON,
+          middleware: (req, res, next, core) => {
```

Comment:
unused variables IDE warnings niet vervelend hier?
```suggestion
          middleware: (req, res) => {
```
of misschien `middleware: (req, res, _next, _core) => {`